.github/workflows/daily.yml

```
@@ -876,7 +876,7 @@ jobs:
   build-macos:
     strategy:
       matrix:
-        os: [macos-12, macos-14]
+        os: [macos-13, macos-14, macos-15]
```

we just need the oldest one and newest one.
```suggestion
        os: [macos-13, macos-15]
```